package/base-files/files/etc/rc.common

```diff
@@ -62,20 +62,36 @@ depends() {
 	return 0
 }
 
+EXTRA_HELP=""
+EXTRA_COMMANDS=""
+extra_command() {
+	local cmd="$1"
+	local help="$2"
+
+	local extra
+
+	extra="$(printf "%-16s%s" "${cmd}" "${help}")"
+
+	EXTRA_HELP="\t${extra}\n${EXTRA_HELP}"
```

Comment:
You are mixing two purposes of function to add commands. One is to append commands to the end (normal usage), one is to prepend commands to appear at the beginning of help text (usage in help() function). The current implementation always prepends the command to the beginning of help text, which is counter-intuitive and looks weird. Maybe you need two functions?

Reply:
I just need a function here that adds the available commands.